Dependencies/napi/CMakeLists.txt

```diff
@@ -44,6 +44,12 @@ if(NAPI_JAVASCRIPT_ENGINE STREQUAL "Chakra")
     target_compile_definitions(napi PUBLIC USE_EDGEMODE_JSRT)
 endif()
 
+if (UNIX AND NOT ANDROID AND NOT APPLE)
+    target_include_directories(napi PUBLIC "/usr/include/webkitgtk-4.0")
+    find_library(JSCORE_LIBRARY JavaScriptCore)
+    target_link_to_dependencies(napi PUBLIC ${JSCORE_LIBRARY})
+endif()
```

Comment:
Doesn't this break the `javascript_engine` injection mechanism below? I think this should only be done if `javascript_engine` isn't already a target, and it should probably be a third case in the conditional that begins on line 58.

Reply:
I moved that block into a new conditional branch. Let  me know if I missed something.